deployments/cluster/educator.nix

```diff
@@ -20,6 +20,8 @@ in
     4040        # Educator HTTP API
   ];
 
+  services.postgresql.enable = true;
```

Comment:
You *have* to set the package explicitly.

Reply:
Why is that?

Reply:
If you don't, nixos can't tell which version you want to use when a new version becomes available and your database is still in the older version.